framework/framework/server.ts

```
@@ -125,8 +125,8 @@
     _id: number;
 }
 
-export interface HandlerCommon { }
-export class HandlerCommon {
+export interface HandlerCommon<C> { } // eslint-disable-line @typescript-eslint/no-unused-vars
```

Biome (1.9.4)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)